docs/installation/setup.rst

```diff
@@ -13,7 +13,7 @@ MetalK8s_ clusters require machines running CentOS_ / RHEL_ 7.6 or higher as
 their operating system. These machines may be virtual or physical, with no
 difference in setup procedure.
 
-For this quickstart, we will need 5 machines (or 3, if running workload
+For this installation, 5 machines are required (or 3, if running workload
 applications on your control-plane nodes).
```

Comment:
Not sure this kind of recommendation (or the one saying we'll need 4 deployments, in the Expansion doc) is still relevant in a generic Installation guide. Worth revamping after all the other tasks are done (I'll tackle this in #2112).